x-pack/plugins/security_solution/cypress/README.md

```diff
@@ -353,6 +401,9 @@ Similar approach should be used in defining all index patterns, rules, and queri
 
 ## Development Best Practices
 
+### Make sure your test fails
```

Comment:
Do we want empty chunks still above this in our README like development best practices? I would maybe remove it for now?